homeassistant/components/bluetooth/passive_update_coordinator.py

```diff
@@ -106,61 +104,53 @@ def __init__(
         ] = {}
         self.update_method = update_method
 
+        self.entity_names: dict[PassiveBluetoothEntityKey, str | None] = {}
         self.entity_data: dict[PassiveBluetoothEntityKey, _T] = {}
         self.entity_descriptions: dict[
             PassiveBluetoothEntityKey, EntityDescription
         ] = {}
         self.devices: dict[str | None, DeviceInfo] = {}
 
         self.last_update_success = True
-        self._last_callback_time: float = NEVER_TIME
-        self._cancel_track_available: CALLBACK_TYPE | None = None
+        self._cancel_track_unavailable: CALLBACK_TYPE | None = None
+        self._cancel_bluetooth_advertisements: CALLBACK_TYPE | None = None
         self._present = False
+        self.last_seen = 0.0
```

Comment:
I changed this from `_last_callback_time` to `last_seen` since many devices will need to override the `available` check in the entity